tests/qgis-projects/tests/form_edition_value_relation_field.md

```diff
@@ -8,41 +8,16 @@ Lizmap plugin installed for QGIS Server is needed
 
 ### Check the default form
 
-* [ ] Click on add a `point` in the *Edition* panel (do not click on the map yet)
-* [ ] Check that the *No expression* available values has **5** options :
-  * [ ] **Zone A1**
-  * [ ] **Zone A2**
-  * [ ] **Zone B1**
-  * [ ] **Zone B2**
-  * [ ] an empty value
-* [ ] Check that the *Simple expression* available values has **3** options :
-  * [ ] **Zone A1**
-  * [ ] **Zone B1**
-  * [ ] an empty value
-* [ ] Check that the *Child field* available values has **1** option :
-  * [ ] an empty value
-* [ ] Check that the *Geom expression* available values has **1** option
-  * [ ] an empty value
+Covered by Cypress
```

Comment:
Why not removing ?
Less thing to read among all these markdown file when doing the manual testing.

Reply:
Because still one part cannot be removed

Reply:
I'm not talking about the file itself.
But the section `### Check the default form` and `### Check the drill-down fields`